libs/shared/src/entities/notification-template/notification-template.interface.ts

```diff
@@ -63,7 +63,7 @@ export interface INotificationTemplateStep {
 
 export interface IMessageFilter {
   isNegated?: boolean;
-  type: BuilderFieldType;
+  type?: BuilderFieldType;
```

Comment:
What's the reason behind making this field optional?

Reply:
When we fetch a template and create a template
this field is not needed to apply filter.
![type-steps](https://github.com/novuhq/novu/assets/39362422/ef4fd06e-3411-4866-83bf-8ec4448cdddb)